docs/content/reference/versioning-support.md

```diff
@@ -1,42 +1,75 @@
-# Versioning support
-
+# Versioning Support
 There are different components that might require independent versioning and support level:
 
-- API `hypershift.openshift.io`.
-- HyperShift Operator (HO).
-- Control Plane Operator (CPO).
-- HyperShift CLI.
+- Management Cluster
+- API `hypershift.openshift.io`
+- HyperShift Operator (HO)
+- Control Plane Operator (CPO)
+- HyperShift CLI
+- Hosted Control Plane (HCP) CLI
+
+## Support Level
+### Management Cluster
+There is no hard requirement on what OCP version the OpenShift management cluster needs to be running.
```

Comment:
Something like this would help my understanding (note that I'm guessing at some of it).
```suggestion
The upstream HyperShift project does not place strict requirements on the OpenShift version of your management cluster. However, different versions of the HO are thoroughly tested only on a limited set of OpenShift versions, and this should inform your deployment decisions.

For a given build of the upstream HO (e.g. [v0.1.39](https://github.com/openshift/hypershift/releases/tag/v0.1.39)), the list of OCP versions tested for Management Cluster compatibility can be found here: `link to file in repository with this information -- ideally where it is programmatically checked and results in warnings if it is not met at runtime`.

For production use & support, it is required to use a downstream product which bundles a supported build of the HyperShift Operator. This downstream product is called [Multi-Cluster Engine](https://docs.openshift.com/container-platform/4.16/architecture/mce-overview-ocp.html) (MCE) and it is available through OpenShift's OperatorHub. 

MCE versions _do_ require specific OCP versions for the Management Cluster in order to remain in a supported. 
Each version documents its own support matrix. For example, 
- MCE 2.5: https://access.redhat.com/articles/7056007 . 
- MCE 2.4: https://access.redhat.com/articles/7027079 . 

As a heuristic, a new release of MCE will run on:
- The latest, yet to be released version of OpenShift.
- The latest GA version of OpenShift.
- 2 versions prior to the latest GA version.

Versions of MCE can also be obtained with the Advanced Cluster Management (ACM) offering. If you are running ACM, refer to product documentation to determine the bundled MCE version.
```